topics/github-api/index.md

```diff
@@ -0,0 +1,9 @@
+---
+created_by: GitHub
+display_name: GitHub API
+github_url: https://developer.github.com/
+related: github, github-client, github-app, api, api-rest, graphql
+short_description: The GitHub API allows you to build applications that integrate with GitHub.
+topic: github-api
+---
+The GitHub API provides endpoints for users to consume GitHub data as well as make changes on a user's behalf. The latest version is v4, the GraphQL API. The GitHub GraphQL API v4 represents an architectural and conceptual shift from the GitHub REST API v3.
```

Comment:
This looks fantastic, @cheshire137. What do you think of adding a few words on what makes the GraphQL API different? Maybe something like:

> The GitHub GraphQL API v4 represents an architectural and conceptual shift from the GitHub REST API v3 and allows for users to craft queries of the exact data they need.